integration/release/error.go

```diff
@@ -0,0 +1,32 @@
+package release
```

Comment:
We only need `WaitForChartStatus` and `WaitForChartInfo`. So moving this in from e2e-harness.